### PR TITLE
Fixed the README file.

### PR DESCRIPTION
**Pull request by RiversideRocks**
Made the Adam video watchable and made fixed the PHP website URL in the README.md file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,7 +61,7 @@ Visit [db_server](https://github.com/gautamkrishnar/nothing-private/tree/master/
 #### Technologies used
 
 * [Client.js Browser fingerprinting](https://github.com/jackspirou/clientjs)
-* [PHP](https://secure.php.net/)
+* [PHP](https://www.php.net/)
```
Comment (gautamkrishnar) [inline README.md:64]: I don't think that this change is required anymore. www subdomain will work now. Can you please revert the change: https://github.com/docker-library/php/pull/809

File: README.md [hunk 2]
```diff
@@ -18,8 +18,9 @@ This project is a proof of concept that any website can identify and track you,
 ## Hey! How?
 
 Hope you are surprised! :smile: Yes, the website can remember your name even if you had visited it via **private browsing** or **incognito mode**. Yes, nothing is private in this world anymore! This is what the big companies are doing with your identity. You think that going into private mode will wipe out all the traces? **Absolutely not!** In reality, using private browsing or incognito mode will just help you to clear your browsing history. Your internet service provider, search engines, and your favorite websites **can still track you**. They know your likes and dislikes. They use your data to earn money. The video below explains everything:
-
+<a href="https://www.youtube.com/watch?v=5pFX2P7JLwA" >
 [![Not free](https://img.youtube.com/vi/5pFX2P7JLwA/0.jpg)](https://www.youtube.com/watch?v=5pFX2P7JLwA)
```
Comment (gautamkrishnar) [inline README.md:22]: This does exactly the same thing as a href. 
```
[Not free](https://img.youtube.com/vi/5pFX2P7JLwA/0.jpg) -> Image 
[![Not free](https://img.youtube.com/vi/5pFX2P7JLwA/0.jpg)](https://www.youtube.com/watch?v=5pFX2P7JLwA) -> Hypherlink to video. 
```

Can you please explain why you had added this change?

**Comment by RiversideRocks** [comment]
Odd, earlier I was getting the camo.githubusercontent.com url. Very sorry!

**Comment by gautamkrishnar** [comment]
@RiversideRocks Its okay thanks

**Pull request closed without merging**